client/.gitignore

```diff
@@ -24,4 +24,6 @@ yarn-debug.log*
 yarn-error.log*
 .eslintcache
 
-env-config.js
\ No newline at end of file
+env-config.js
+
+17'
```

Comment:
I'm not sure where the file `17'` comes from. It will generated when I execute `mvn clean install`. Whatever let's ignore it by adding it to `.gitignore`.